src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -43,6 +43,46 @@ static constexpr Property<bool, PropertyMutability::RW> exclusive_async_requests
  */
 static constexpr Property<std::string, PropertyMutability::WO> config_device_id{"CONFIG_DEVICE_ID"};
 
+/**
+ * @brief Allow low precision transform
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<bool, PropertyMutability::RW> lp_transforms_mode{"LP_TRANSFORMS_MODE"};
+
+/** @cond INTERNAL */
+inline std::ostream& operator<<(std::ostream& os, const ov::threading::IStreamsExecutor::ThreadBindingType& type) {
+    switch (type) {
+    case ov::threading::IStreamsExecutor::NONE:
+        return os << "NONE";
+    case ov::threading::IStreamsExecutor::CORES:
+        return os << "CORES";
+    case ov::threading::IStreamsExecutor::NUMA:
+        return os << "NUMA";
+    case ov::threading::IStreamsExecutor::HYBRID_AWARE:
+        return os << "HYBRID_AWARE";
+    default:
+        OPENVINO_THROW("Unsupported thread binding type value");
+    }
+}
+
+inline std::istream& operator>>(std::istream& is, ov::threading::IStreamsExecutor::ThreadBindingType& type) {
```

Comment:
we have public property `Affinity`, can we use it instead of `ThreadBindingType` ?
@wangleis

Reply:
Affinity is public property and ThreadBindingType is ov::threading::IStreamsExecutor::ThreadBindingType enum class.
I forgot why I had added above 2 operator function, seem not need them,  let me check whether can be remove them.